### ISSUE TITLE
[css-transforms-1] Add accessibility section

### ISSUE THREAD

**Issue by IanPouncey** [w3c/csswg-drafts#3870]
The CSS Accessibility task force would like an accessibility guidance section added to this module.

The guidance would relate to WCAG 2.0/1 [Success Criterion 2.2.2 Pause, Stop, Hide](https://www.w3.org/TR/WCAG21/#pause-stop-hide), indicating that in some circumstances a mechanism for pausing, stopping, or hiding moving content must be provided.

I think it will be possible to write standard guidance that can be applied to any module that enables movement or animation and the task force would be happy to help draft this.

**Comment by birtles** [comment]
I wonder, would it not be better to add this to CSS Transitions, CSS Animations, and Web Animations instead? CSS Transforms by itself does not produce moving content.

**Comment by tabatkins** [comment]
Yes, that's my thought. Nothing in Transforms "enables" motion any more than 'margin' does; transforms are just a (static) layout tool unless you apply an animation.

**Comment by IanPouncey** [comment]
The reason for including it here (as well as everywhere else you mention) is that the module does include examples of [animation using transforms with SVG](https://www.w3.org/TR/css-transforms-1/#svg-animation). I agree that it's not necessary to include this note in every module that includes a property that can be animated, but there is a specific case covered in css-transforms-1.

**Comment by tabatkins** [comment]
Ah, yeah, that's reasonable. I'd forgotten that section was even in there.

**Comment by dirkschulze** [comment]
I think we should remove this section entirely. There is not UA that implements animation of the `transform` attribute/property via `<animate>` or `<set>` nor did I hear any interest from UAs to do the implementation.

To animate transforms with SMIL, users may choose `<animateTransform>` which gets covered by a different specification [SVG Animations](https://svgwg.org/specs/animations/).

**Comment by dirkschulze** [comment]
As discussed during the CSS WG meeting: We did agree that there should be text in the CSS Transitions spec (which already has a section) and CSS Animations spec with a description of the issue and a reference to the accessibility guidance. If possible, the text should be easily accessible from all motion affected properties (for instance when following the link for "Animation type" that is already set on all CSS properties in updated specifications).

We also acknowledge that UAs should provide an option to mute/pause/slow down motions and will work on it as part of the mentioned specification.

With the removal of the SVG Animations sections from CSS Transforms we agreed that there is nothing to do for CSS Transforms right now.
